pages/components/search-panel/index.jsx

```diff
@@ -28,7 +38,7 @@ class SearchPanel extends Component {
 
           <div className="govuk-grid-column-one-third">
             <div className="view-all-link">
-              <a href={`/search/${searchType}`}>
+              <a href={`/search/${searchType}?${searchableModels.find(m => m.name === searchType).query}`}>
```

Comment:
I don't see where the `query` property is set. Am I missing something? It looks like it will always be undefined.